src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+
+    public void doWork() {
```

Comment:
fix that in the whole project
```suggestion
    @Override
    public void doWork() {
```